package.json

```diff
@@ -22,11 +22,11 @@
     "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run format-check && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
     "prepare": "husky install",
     "format-check": "prettier --check . && eslint .",
-    "format-fix": "prettier --write . && eslint . --fix",
+    "format-fix": "prettier --write .",
```

Comment:
Should we make this part of the build command?

Reply:
no, the whole point of the commit hook is that it's supposed to automatically format the code for you

Reply:
hmm this is an accidental delete while I tried to test things